backend/protocol_rpc/message_handler/types.py

```diff
@@ -1,17 +1,33 @@
+from enum import Enum
 from dataclasses import dataclass
 
-from backend.protocol_rpc.types import EndpointResult
+
+class EventType(Enum):
+    DEBUG = "debug"
+    INFO = "info"
+    SUCCESS = "success"
+    ERROR = "error"
```

Comment:
shouldn't we use event types from `loguru` or `logging` packages?

Reply:
loguru doesn't have actual types that I can import it seems, rather they have pre-defined and customizable levels but I don't think I can simply import the type without writing some logic - happy to be wrong tho